plugins/services/src/js/containers/services/ServicesContainer.js

```diff
@@ -144,10 +143,6 @@ class ServicesContainer extends React.Component {
   componentDidMount() {
     DCOSStore.addChangeListener(DCOS_CHANGE, this.onStoreChange);
 
-    // Don't block the whole screen if Mesos state is not there
```

Comment:
This seems unrelated?

Reply:
I looked back to the commit where I added that `getResources` method back then. There I added this event listener. It was required since we haven't had stream and tended to use `MesosStateStore` to calculate resources as use can see from the method I deleted